components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -144,6 +144,8 @@
 <config_Redi_N2_based_taper_enable>.true.</config_Redi_N2_based_taper_enable>
 <config_Redi_N2_based_taper_min>0.1</config_Redi_N2_based_taper_min>
 <config_Redi_N2_based_taper_limit_term1>.true.</config_Redi_N2_based_taper_limit_term1>
+<config_Redi_use_quasi_monotone_limiter>.false.</config_Redi_use_quasi_monotone_limiter>
```

Comment:
@vanroekel, I would like to change this to true, so the quasi-monotone limiter is on by default for all simulations. Do you agree?

Reply:
yes, definitely.  that should be the default